index.html

```diff
@@ -0,0 +1,199 @@
+              global sporting event, racing in 22 countries across 5 continents.
+            </p>
+          </div>
+          <div class="venue-details">
+            <p class="summit-date">2022.10.15(Thu) ~ 16(Fri)</p>
+            <p class="summit-venue">
+              @ Silverstone, Home of British Motor Racing.
+            </p>
+          </div>
+        </div>
+      </section>
+      <section class="program" id="main-program">
+        <div class="section-header">
+          <h2>main program</h2>
+          <hr class="orange-line" />
+        </div>
+        <div class="program-wrapper inner-content">
+          <div class="program-tile">
+            <i class="fa-solid fa-chalkboard-user"></i>
+            <h3>lecture</h3>
+            <p>
+              Listen to the speakers from various countries about the messages
+              of sharing and opening
+            </p>
+          </div>
+          <div class="program-tile">
+            <i class="fa-solid fa-chalkboard"></i>
+            <h3>f1 exhibition</h3>
+            <p>
+              Appreciate various drivers applying for their F1 driver's
+              licences, organized by Silverston circuit
+            </p>
+          </div>
+          <div class="program-tile">
+            <i class="fa-solid fa-message"></i>
+            <h3>Forum</h3>
+            <p>
+              Have time for attendees to share their thoughts and opinions with
+              F1 drivers, and the conveference committee
+            </p>
+          </div>
+          <div class="program-tile">
+            <i class="fa-solid fa-users"></i>
+            <h3>Workshop</h3>
+            <p>
+              F1 sponsors showcase some of the sustainable products they are
+              developing
+            </p>
+          </div>
+          <div class="program-tile">
+            <i class="fa-solid fa-spinner"></i>
+            <h3>f1 ignite</h3>
+            <p>
+              Get opportunities to network with F1 affiliates around the world,
+              also after the summit.
+            </p>
+          </div>
+        </div>
+        <a href="#" class="join-button">Join F1 Sustainability Effort</a>
+        <a href="#" class="view-more">see the whole program</a>
+      </section>
+      <section class="featured-speakers" id="speakers">
+      </section>
+      <section class="event-partners" id="partners">
+        <div class="section-header">
+          <h2>Partners</h2>
+          <hr class="orange-line" />
+        </div>
+        <div class="partners-wrapper inner-content">
+          <img src="./images/formula-1-logo.png" class="partner-logo" alt="" />
+          <img src="./images/kakao.png" class="partner-logo" alt="" />
+          <img src="./images/naver.png" class="partner-logo" alt="" />
+          <img src="./images/google.png" class="partner-logo" alt="" />
+          <img src="./images/airbnb.png" class="partner-logo" alt="" />
+        </div>
```

Comment:
- [ ] Kindly add alt attributes to images and please do not leave them blank. Keep coding 💪